File: 344-ReverseString/344-ReverseString.py
# Last updated: 7/8/2026, 4:48:08 PM
class Solution:
    def reverseString(self, s: List[str]) -> None:
        l = 0 
        r = len(s)-1
        while l<r:
            s[l],s[r]=s[r],s[l]
            l+=1
            r-=1        

File: 344-ReverseString/344-ReverseString.java
// Last updated: 7/8/2026, 4:48:13 PM
class Solution {
    public void reverseString(char[] s) {
        // char[] a = new char[s.length];
        
        int j = 0;
        for(int i=s.length-1;i>j;i--,j++){
            char b = s[j];
            s[j] = s[i]; 
            s[i] = b;
            
        }
        
    }}